### DISCUSSION TITLE
How to customize the table filter reset action?

### DISCUSSION THREAD

**Discussion by devhoussam1998** [filamentphp/filament#12099] [Help]
### Package

Table builder

### Package Version

v3.2.61

### How can we help you?

I want to customize the table filter reset action

<img width="267" alt="Screenshot 2024-03-29 at 17 52 31" src="https://github.com/filamentphp/filament/assets/121255432/d761e38b-e89f-4ca4-b6b5-c922d09ce44d">

I tried https://filamentphp.com/docs/3.x/tables/filters/getting-started#customizing-the-filters-trigger-action but does not achieve my desire.

**Comment by VincvanAs** [comment]


**Reply by devhoussam1998** [reply]
My Question is clear I want to customize the table filter reset action?

Example:

instead of `->link()` I want to use `->button()` to reset filter action.

**Reply by Saniori0** [reply]
What wrong with u man?

**Comment by bikash-s-bhandari** [comment]
how to change the position of reset button in table filter?

**Reply by leandrocfe** [reply]
https://github.com/filamentphp/filament/discussions/10080#discussioncomment-9224308
